### ISSUE TITLE
Unable to parse money in Euro

### ISSUE THREAD

**Issue by crbelaus** [kipcole9/money#95]
I am trying to save a money field from a Phoenix input. Everything works properly except for the validation when I use Euros.

I've traced the problem to the `Money.parse` function.

```
iex(1)> Money.parse("99.99$")
#Money<:USD, 99.99>
iex(2)> Money.parse("99.99€")
{:error,
 {Money.Invalid,
  "Unable to create money from <<130, 172>> and <<57, 57, 46, 57, 57, 226>>"}}
```

It looks that the `€` symbol is correctly associated with the `EUR` currency:

```
iex(3)> {:ok, currencies} = Cldr.Currency.currency_strings "en", Money.Cldr
{:ok, %{...}}
iex(4)> currencies["€"]
:EUR
iex(5)> currencies["$"]
:USD
```

It may be some trouble with the regular expression that splits the amount and the currency, since as you can see in the first error, it is interpreting the `€` as `<<130, 172>>`.

**Comment by kipcole9** [comment]
Apologies for the inconvenience and thanks for the report.  I have [fixed on master](https://github.com/kipcole9/money/releases/tag/v3.2.3) and published [version 3.2.3](https://hex.pm/packages/ex_money/3.2.3) to hex.

The issue was that I wasn't setting the `:unicode` flag on the regex used to parse the currency string (pretty much as you suspected).

**Comment by crbelaus** [comment]
Wow, that is an incredibly fast fix.

Thank you for your fast response and for such a wonderful package!! :raised_hands:
